nautobot/dcim/management/commands/populate_platform_network_driver.py

```diff
@@ -0,0 +1,86 @@
+from django.core.management.base import BaseCommand
+from netutils.lib_mapper import NAPALM_LIB_MAPPER
+
+
+from nautobot.dcim.models import Platform
+
+
+HELP_TEXT = """
+Populate the Platform.network_driver field from the napalm_driver or slug fields.
+
+This command will attempt to find a valid mapping from the Platform.napalm_driver field to a matching entry in netutils.lib_mapper.NAPALM_LIB_MAPPER.
+
+If no mapping is found, the value of the Platform.slug field will be used instead.
+
+--interactive can be used to prompt for confirmation before making changes to each Platform.
+
+--no-use-slug-field can be used to disable populating the network_driver field from the Platform.slug field.
+
+--no-use-napalm-driver-field can be used to disable populating the network_driver field from the Platform.napalm_driver field.
+
+By default, the network_driver field will only be populated if it is currently empty. Use --force to overwrite existing values.
+"""
+
+
+class Command(BaseCommand):
+    help = HELP_TEXT
+
+    def add_arguments(self, parser):
+        group = parser.add_mutually_exclusive_group()
```

Comment:
This makes sure that `--no-use-slug-field` and `--no-use-napalm-driver-field` arguments cannot be used together right?

Reply:
nvm stupid question ofc it is.